### PR TITLE
filer drag and drop upload modal updates

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

screen shot 2016-02-29 at 14 21 33 2
screen shot 2016-02-29 at 14 20 08
screen shot 2016-02-29 at 14 20 09

### CODE REVIEW

File: filer/private/sass/components/_drag-and-drop.scss [hunk 1]
```diff
@@ -334,15 +334,16 @@ form .form-row {
...
     background: $white;
     box-shadow: $base-box-shadow;
     .icon {
-        font-size: 30px;
+        font-size: 35px;
+        color: $color-primary;
     }
     .text {
-        margin: 15px 0 10px 0;
+        margin: 5px 0 10px 0;
```
Comment (dhorelik) [inline filer/private/sass/components/_drag-and-drop.scss]: `margin: 5px 0 10px;`

Comment (Brankub) [inline filer/private/sass/components/_drag-and-drop.scss]: last zero could be removed

**Comment by Brankub** [comment]
:+1:

**Comment by dhorelik** [comment]
lgtm

File: filer/static/filer/js/addons/table-dropzone.js [hunk 2]
```diff
@@ -32,7 +33,9 @@
         var baseUrl;
         var baseFolderTitle;
         var updateUploadNumber = function () {
+            uploadText.addClass('hidden');
             uploadNumber.text(maxSubmitNum - submitNum + '/' + maxSubmitNum);
+            uploadText.removeClass('hidden');
```
Comment (sergii-kulakovskyi) [inline filer/static/filer/js/addons/table-dropzone.js]: why do you need to add and then remove class immediately?

Comment (lory87) [inline filer/static/filer/js/addons/table-dropzone.js]: no need for it removed and tested

Comment (dhorelik) [inline filer/static/filer/js/addons/table-dropzone.js]: my guess is that text update is slow, so we hide it until it's updated. But `.text()` shouldn't be that slow actually?

File: filer/private/sass/components/_drag-and-drop.scss [hunk 3]
```diff
@@ -362,10 +366,34 @@ form .form-row {
...
 
 .filer-dropzone-cancel {
-    margin-top: 15px;
+    padding-top: 10px;
+    border-top: solid 1px $gray-lighter;
+    margin: 15px -15px 10px;
+    a {
+        font-size: $font-size-small;
+        color: $gray !important;
+        cursor: pointer;
```
Comment (dhorelik) [inline filer/private/sass/components/_drag-and-drop.scss]: `a { cursor: pointer; }` happens anyway, no?

File: filer/templates/admin/filer/folder/directory_table.html [hunk 4]
```diff
@@ -175,14 +178,14 @@
         </div>
 
         <div class="js-filer-dropzone-cancel filer-dropzone-cancel hidden">
-            <a href="#"><span class="fa fa-times"></span> Cancel upload</a>
+            <a href="#"></span>{% trans "cancel" %}</a>
```
Comment (dhorelik) [inline filer/templates/admin/filer/folder/directory_table.html]: `</span>`

**Comment by dhorelik** [comment]
lgtm

**Pull request merged**